### DISCUSSION TITLE
Deleted github repositories

### DISCUSSION THREAD

**Discussion by GeekkyCoder** [community/community#35515] [New to GitHub]
guys i have mistakeny deleted the reposiotries how to get them back, was working on hactoberfest 2022 and had sucessfully 2 merged pull requests now my 2 merged requests are exludeed from hactoberfest! plz help

**Comment by radar-blip** [comment]
You'll have to ask support to get them back for you: [Support Page](https://support.github.com/request) >> 'Restore deleted data'

**Comment by DSDmark** [comment]
# It may help you. Try these step:-

- first go to setting option :- 
![Screenshot from 2022-10-08 14-55-08](https://user-images.githubusercontent.com/85744816/194700434-2a1a2823-5ebe-4785-b363-90e9d65fabf8.png)

- and inside there is the section ```Code, planning, and automation``` in that section, click on **repository** option 
![Screenshot from 2022-10-08 14-56-47](https://user-images.githubusercontent.com/85744816/194700600-71a9d589-8166-44e0-81f5-63250660604e.png)

- After that, click on deleted repository option.
![Screenshot from 2022-10-08 14-57-04](https://user-images.githubusercontent.com/85744816/194700661-86001adc-760e-4d13-a0af-f1abdcbddc73.png)

- You can see all of your **deleted repository** you restore it by click on restore button

**Comment by jfullstackdev** [comment]
it can easily be retrieved as was demonstrated on top
